ironweb.d.ts

```diff
@@ -146,10 +146,10 @@ export interface UserCreateOptions {
 /**
  * SDK Namespaces
  */
-
 export interface User {
     deauthorizeDevice(): Promise<{transformKeyDeleted: boolean}>;
     changePasscode(currentPasscode: string, newPasscode: string): Promise<void>;
+    rotateCurrUsersPrivateKey(passcode: string): Promise<void>;
```

Comment:
I don't think there's any reason to put `CurrUsers` since these operations are always acting for the current use. Something like `rotateMasterKey` would be better.